[PATCH] dts: mt7988: Remove dtb path in Makefile

[Description]
Remove dtb path in Makefile since we don't need to specify
dtb paths in newer Uboot.

Change-Id: If911783695d1699339060e08ff9d87fe5b6f6282
---
 arch/arm/dts/Makefile | 9 ---------
 1 file changed, 9 deletions(-)

Change-Id: If911783695d1699339060e08ff9d87fe5b6f6282
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -1135,12 +1135,6 @@ dtb-$(CONFIG_ARCH_MEDIATEK) += \
 	mt7986b-sd-rfb.dtb \
 	mt7986a-emmc-rfb.dtb \
 	mt7986b-emmc-rfb.dtb \
-	mt7988-rfb.dtb \
-	mt7988-snfi-nand-rfb.dtb \
-	mt7988-spim-nand-rfb.dtb \
-	mt7988-spim-nor-rfb.dtb \
-	mt7988-emmc-rfb.dtb \
-	mt7988-sd-rfb.dtb \
 	mt8183-pumpkin.dtb \
 	mt8512-bm1-emmc.dtb \
 	mt8516-pumpkin.dtb \
